omnibus/files/private-chef-cookbooks/private-chef/templates/default/opscode-chef-mover.config.erb

```diff
@@ -99,8 +99,7 @@
                   {s3_parallel_ops_fanout, <%= node['private_chef']['opscode-erchef']['s3_parallel_ops_fanout'] %>}
                  ]},
   {stats_hero, [
-               %% Set sender pool size to DB max_connections to avoid contention
-               {udp_socket_pool_size, <%= node['private_chef']['postgresql']['max_connections'] %> },
+               {udp_socket_pool_size, <%= @udp_socket_pool_size %> },
```

Comment:
I'm wondering if it would be better to set the default value of `udp_socket_pool_size` to `nil` in the attribute file and then here do: `<%= @db_pool_size || @udp_socket_pool_size -%>`. This would mean that the UDP socket pool size would default to the db_pool_size which I think makes sense since the goal of the old default was to ensure that we are never waiting on a udp socket worker and since request concurrency is pretty naturally limited by the db_pool_size.